tests/CommandsTest.php

```diff
@@ -167,7 +167,9 @@
         mkdir($dir, 0777, true);
     }
 
-    pest()->artisan('tenancy:install');
+    pest()->artisan('tenancy:install')
+        ->expectsConfirmation('Would you like to show your support by starring the project on github ?', 'no')
```

Comment:
```suggestion
        ->expectsConfirmation('Would you like to show your support by starring the project on GitHub?', 'no')
```

Reply:
@stein-j You marked this as resolved but didn't fix it?

Reply:
Yes I did in this commit a8f416dc1a3ccb6c0050d6545acf088da10aaeec
Did I use the Github feature wrong ?

Reply:
https://github.com/archtechx/tenancy/pull/968/files#diff-2d46e8828aea0422777576ca8e903262bf80a262978ec02ad3d0783c38fe1b32R171

Not sure what you mean by "Github feature". If you want to directly commit my changes you can just use this:

<img width="175" alt="CleanShot 2022-10-18 at 6 29 17@2x" src="https://user-images.githubusercontent.com/33033094/196489027-b9a6aafb-4274-4c2a-836a-b452940b906b.png">

Reply:
Anyway, I fixed that now.